src/FlaUI.Adapter.White/Mappings/ControlDictionaryItems.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FlaUI.Core.Definitions;
+
+namespace TestStack.White.Mappings
+{
+    public class ControlDictionaryItems : List<ControlDictionaryItem>
+    {
+        public virtual void AddPrimary(Type testControlType, ControlType controlType)
+        {
+            Add(ControlDictionaryItem.Primary(testControlType, controlType));
+        }
+
+        public virtual void AddSecondary(Type testControlType, ControlType controlType)
+        {
+            Add(ControlDictionaryItem.Secondary(testControlType, controlType));
+        }
+
+        public virtual void AddPrimary(Type testControlType, ControlType controlType, bool hasPrimaryChildren)
```

Comment:
What is the boolean `hasPrimaryChildren` for?

Reply:
fixe